pkg/discovery/mdns.go

```diff
@@ -20,11 +17,17 @@ import (
 var log = logger.NewLogger("dapr.runtime.discovery")
 
 // RegisterMDNS uses mdns to publish an entry of the service to a local network
-func RegisterMDNS(id string, port int) error {
+func RegisterMDNS(id string, host string, ips []string, port int) error {
```

Comment:
why do we need to pass host ? line 22 will overwrite host with os.hostname().

Reply:
No we don't good spot - this was left from an earlier usage.